lib/filter-blocked.js

```diff
@@ -1,25 +1,54 @@
 const async = require('async')
+const pull = require('pull-stream')
 const pullFilterAsync = require('./pull-filter-async')
+const paramap = require('pull-paramap')
 
 module.exports = function (blockSources, { isBlocking, checkRoot = false, useRootAuthorBlocks = false }) {
-  return pullFilterAsync((msg, cb) => {
-    var checks = []
-    blockSources.forEach(source => {
-      if (msg.value) {
-        checks.push({ source, dest: msg.value.author })
-      }
-      if (checkRoot && msg.root && msg.root.value) {
-        checks.push({ source, dest: msg.root.value.author })
+  // Maybe hacky but it seems true in practice
+  const me = blockSources[0]
```

Comment:
Could we use `api.keys.sync.id()` or something similar here instead?

Reply:
I thought about doing that, but there's no `api` object access in this file and I don't know how to get that.

Reply:
Got a good solution here?

Reply:
Nope, I think this is probably as good as we'll get -- the dichotomy between `lib` and `modules` kills me, so much code that can't be shared!